TestingScripts/CommonTasks/Components.json

```diff
@@ -0,0 +1,38 @@
+{
+    "Bentley":       {"dll": "iTwinBentley.dll", "exe": "BentleyTest",
+                           "ignore":["iModelCore", "Bentley","Tests"],
+                           "pdb": ["Bentley", "BentleyDll", "iTwinBentley.pdb"],
+                           "product": "Bentley-Gtest"},
+    "BeSQLite":      {"dll": "iTwinSQLite.dll", "exe": "BeSQLiteTest",
```

Comment:
Is this possible that we can generalize this information (may be by using pattern matching) rather than hardcoding it?

Reply:
ok